personality/wn/rpms/config.pan

```diff
@@ -7,7 +7,11 @@ unique template personality/wn/rpms/config;
 variable HEP_OSLIBS ?= true;
 "/software/packages" = {
     if (is_boolean(HEP_OSLIBS) && HEP_OSLIBS) {
-        SELF[escape('HEP_OSlibs_SL6')] = nlist();
+        if (OS_VERSION_PARAMS['major'] == 'sl5') {
+            SELF[escape('HEP_OSlibs_SL5')] = nlist();
+        } else { 
+            SELF[escape('HEP_OSlibs_SL6')] = nlist();
+        };
```

Comment:
Sounds good if the name "wlcg" makes sense to everyone, the main goal is to reduce duplication across the templates.

`features/hep_oslibs/config` might be clearer unless you plan to put more general wlcg configuration in there as well.